tests/integration/mixer/policy/ratelimiting_test.go

```diff
@@ -255,11 +310,18 @@ func TestMain(m *testing.M) {
 	framework.
 		NewSuite("mixer_policy_ratelimit", m).
 		RequireEnvironment(environment.Kube).
-		SetupOnEnv(environment.Kube, istio.Setup(&ist, nil)).
+		SetupOnEnv(environment.Kube, istio.Setup(&ist, setupConfig)).
 		Setup(testsetup).
 		Run()
 }
 
+func setupConfig(cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+	cfg.Values["pilot.env.PILOT_ENABLE_FALLTHROUGH_ROUTE"] = "true"
```

Comment:
@howardjohn do we still need this? I seem to recall you getting rid of occurrences this a month or two back.

Reply:
Please let me know if I need to change this..